css/style.css

```diff
@@ -0,0 +1,184 @@
+@import url('https://fonts.googleapis.com/css?family=Open+Sans&display=swap');
```

Comment:
Going to paste the directly the files should be moved under packages/react-app/src